### PR TITLE
Silence warnings

### PR DESCRIPTION
**Pull request by pmillspaugh**
Sibling PR to https://github.com/membrane-io/mnode/pull/243

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -114,6 +114,9 @@ unused_lifetimes = "warn"
 trivial_casts = "allow"
 unused_qualifications = "allow"
 
+# MEMBRANE: added to suppress warning
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(web_sys_unstable_apis)'] }
+
```
Comment (juancampa) [inline Cargo.toml:119]: Nice! I didn't know you could suppress warnings here 👍

File: crates/egui_extras/src/syntax_highlighting.rs [hunk 2]
```diff
@@ -37,6 +31,13 @@ pub fn highlight(ctx: &egui::Context, theme: &CodeTheme, code: &str, language: &
     })
 }
 
+// MEMBRANE: moved impl out of highlight fn to suppress warning
+impl egui::util::cache::ComputerMut<(&CodeTheme, &str, &str), LayoutJob> for Highlighter {
+    fn compute(&mut self, (theme, code, lang): (&CodeTheme, &str, &str)) -> LayoutJob {
+        self.highlight(theme, code, lang)
+    }
+}
+
```
Comment (juancampa) [inline crates/egui_extras/src/syntax_highlighting.rs]: Hmm interesting, I don't see any warnings for this code

File: Cargo.toml (hunk 1 above)
Comment (pmillspaugh) [reply Cargo.toml:119]: The Rust compiler had a helpful message after the warning suggesting it :)

File: crates/egui_extras/src/syntax_highlighting.rs (hunk 2 above)
Comment (pmillspaugh) [reply crates/egui_extras/src/syntax_highlighting.rs]: You mean before the change you don't see this (screenshot from vscode, but also seeing on the command line)?
Screenshot 2024-09-04 at 6 20 29 PM

**Comment by pmillspaugh** [comment]
Taco truck

**Pull request merged**